apis/pipelines/v1alpha5/conversion.go

```diff
@@ -112,3 +112,17 @@ func setProviderAnnotation(provider string, resource *v1.ObjectMeta) {
 func removeProviderAnnotation(resource v1.Object) {
 	delete(resource.GetAnnotations(), ResourceAnnotations.Provider)
 }
+
+func convertProviderAndIdToHub(providerAndId ProviderAndId) hub.ProviderAndId {
+	return hub.ProviderAndId{
+		Name: providerAndId.Provider,
+		Id:   providerAndId.Id,
+	}
+}
+
+func convertProviderAndIdToV1Alpha5(providerAndId hub.ProviderAndId) ProviderAndId {
+	return ProviderAndId{
+		Provider: providerAndId.Name,
+		Id:       providerAndId.Id,
+	}
```

Comment:
I think these functions don't need to reference hub and v1alpha5 because they are within the context of v1alpha5 already, so they should just follow the conversion's interface naming of `From` and `To`.

Reply:
Renamed in https://github.com/sky-uk/kfp-operator/pull/384/commits/671ce99fdc84d1fea0ade9f8bac0e6bc44d2ce05